openedx_lti_tool_plugin/tests/test_admin.py

```diff
@@ -25,12 +25,14 @@ def test_instance_attributes(self):
         self.assertEqual(self.admin.search_fields, ['id', 'uuid', 'platform_id', 'client_id', 'subject_id'])
 
 
-class TestCourseAccessConfigurationAdmin(TestCase):
-    """Test course access configuration admin functionality."""
+class TestLtiToolConfigurationAdmin(TestCase):
+    """Test LTI tool configuration admin functionality."""
 
     def setUp(self):
         """Test fixtures setup."""
-        self.admin = CourseAccessConfigurationAdmin(CourseAccessConfiguration, AdminSite)
+        LtiToolConfiguration.objects.all().delete()
```

Comment:
@Serafin-dev Why is it necessary to delete all instances before executing the tests?, shouldn't the queryset be empty already before executing the tests?